### ISSUE TITLE
Hyposphagma

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10905]
Preferred term label:
Hyposphagma

Synonyms
Subconjunctival hemorrhage, Subconjunctival bleeding

Definition (free text, please give PubMed ID)
A condition characterized by bleeding underneath the conjunctiva, often appearing as a well-demarcated red patch on the sclera. It is usually asymptomatic and self-limiting but may be associated with trauma, systemic hypertension, anticoagulant use, or sudden increases in venous pressure. 

Parent term (use hpo.jax.org/app)
Abnormal conjunctiva morphology (HP:502)

Diseases characterized by this term
Traumatic subconjunctival hemorrhage, Hypertensive retinopathy, Coagulopathies

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by daboe01** [comment]
sorry, i missed 11896
